### ISSUE TITLE
Strange Setting Icon Postitioning

### ISSUE THREAD

**Issue by DracoClaw** [ploogins/monaco#2]
If "collapse sidebar" is on, and you have the "Settings icons" theme, the icons get pushed too far to the left. Is it possible to have them all align up with where they are when not in the monaco page?

![image](https://user-images.githubusercontent.com/34274162/98454249-202d7780-216b-11eb-958f-6a8aa106833d.png)
![08-11-20-653](https://user-images.githubusercontent.com/34274162/98454276-7ac6d380-216b-11eb-9ef4-db5f4658d985.gif)

**Comment by AAGaming00** [comment]
yeah, think it's one of the ``flex`` props in che css being wierd
